src/Microsoft.IdentityModel.Protocols.SignedHttpRequest/GlobalSuppressions.cs

```diff
@@ -12,3 +12,4 @@
 [assembly: SuppressMessage("Globalization", "CA1307:Specify StringComparison", Justification = "Vendored component", Scope = "module")]
 [assembly: SuppressMessage("Design", "CA1001:Types that own disposable fields should be disposable", Justification = "Breaking change", Scope = "type", Target = "~T:Microsoft.IdentityModel.Protocols.SignedHttpRequest.SignedHttpRequestHandler")]
 [assembly: SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "Previously released as read / write", Scope = "member", Target = "~P:Microsoft.IdentityModel.Protocols.SignedHttpRequest.SignedHttpRequestInvalidNonceClaimException.PropertyBag")]
+[assembly: SuppressMessage("Globalization", "CA1305:Specify IFormatProvider", Justification = "Breaking Change", Scope = "member", Target = "~T:Microsoft.IdentityModel.Protocols.SignedHttpRequest.SignedHttpRequestHandler)")]
```

Comment:
>[assembly: SuppressMessage( [](http://example.com/codeflow?start=0&length=27)

why is this needed?

Reply:
There's a couple of methods from the Microsoft.IdentityModel.Protocols.SignedHttpRequest project that were throwing error code [CA1305](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1305). After some investigation, it essentially was that we should try and define a certain format a StringBuilder obj that's being created from headers information within the methods so, doing that did not seem to be very useful in those cases as they deal with headers, and we shouldn't format those. 

I opted for suppresing this error instead as it is a currently working functionality.

IFormatProvider doc: https://learn.microsoft.com/en-us/dotnet/api/system.iformatprovider?view=net-7.0

Reply:
sounds good. thanks for the addtional context.